regression/goto-harness-multi-file-project/static_symbols_referencing/first_one.c

```diff
@@ -0,0 +1,13 @@
+#include "first_one.h"
+#include <assert.h>
+
+static int static_function(int a)
+{
+  assert(a == 0);
```

Comment:
Sorry to be annoying, but in the interest of having good tests it might actually make sense to have a couple more assertions here. This is reachable because `non_static_function` is, but to contrast I think it’d be good to also have another static function with this signature that"s called by a non-static function with a _different_ signature whose assertion we don’t reach, demonstrating that we currently do not consider these functions for goto-harness.

Reply:
Done.